cpu/kinetis_common/include/periph_cpu.h

```diff
@@ -322,6 +330,7 @@ typedef struct {
     volatile uint32_t *scgc_addr; /**< Clock enable register, in SIM module */
     uint8_t scgc_bit;             /**< Clock enable bit, within the register */
     uint8_t mode;                 /**< UART mode: data bits, parity, stop bits */
+    uint8_t type;                 /**< Hardware module type (KINETIS_UART or KINETIS_LPUART)*/
```

Comment:
Is there any benefit in doing this in two steps, other than bureaucratic? This CPU supports both UART and LPUART, the KW41Z (other PR) supports only LPUART. The already supported kinetis CPUs support AFAIK, only UART. The LPUART should probably be the default when possible because it is working in STOP and VLPS modes (async operation), to allow lower power consumption.